### PR TITLE
Interactive Documentation for Jupyter-Widget

### PR DESCRIPTION
**Pull request by Harpreet1999**
Added a new jupyter notebook showcasing the way to run tardis including a bokeh spectrum.
## Description

In concern with the current objective to make a new jupyter widget that plots a bokeh spectrum with interactivity of zooming and panning and additional feature to show different individual packets seperately, a jupyter notebook is embedded in the documentation showcasing the use of bqplot for plotting the spectrum and enabled with various GUI buttons.

simulation_u dated
## Motivation and Context

Displays a way to plot spectrum using bqplot and addition of user interface by panning and zooming using scroll.

## How Has This Been Tested?
 the jupyter notebook was embedded in the documentation using sphinx documentation generator and was tested running the same 

Index.rst file in tardis/docs was editted to add the notebook.

## Screenshots (if appropriate):
Screenshot from 2020-03-18 22-45-03

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/running/index.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ A quick start guide to run TARDIS can be found
     :maxdepth: 2
     :caption: Running TARDIS and interactivity
 
+    Widget_Documentation
```
Comment (jaladh-singhal) [inline docs/running/index.rst]: For the sake of ease in typing/finding files (esp. in linux file system), we usually keep filenames all in small letters, with hyphen for spaces. (It's ok for now, but I just wanted to inform you this for your future development projects).

Comment (Harpreet1999) [reply docs/running/index.rst]: Sure. I understand that

**Comment by Harpreet1999** [comment]
> View / edit / reply to [this conversation](https://app.reviewnb.com/tardis-sn/tardis/pull/1091/discussion/) on ReviewNB
> 
> _jaladh-singhal commented on 2020-03-16T17:02:23Z_
> ----------------------------------------------------------------
> 
> Why have you used `global` keyword before figure object in your button trigger functions?
> 
> Also I would suggest to make call `bq.Figure()` in these functions itself, rather than assigning the returned figure object to global variables `fig_1 , 2, 3, etc.` and then accessing them locally. Variables assignment in advance, is needed when you use same variable in multiple functions - it'll be better to refactor your code!


I have been refactoring the code and I've removed all the redundency in the code. 
I'm able to plot different marks even without using the javascript functions I use before to execute the cells below..
Code Is quite optimized and I'm currently working on addition of new features to interactivity. I will soon make a PR. I just want to let you know.

**Comment by Harpreet1999** [comment]
About the zooming and panning I've implemented two features for that.

- Zooming in just X direction with srolling after hovering normally over the graph.

- Zooming in both X and Y direction after enabling  the pan toogle button (using scroll and mouse controls).

**Pull request closed without merging**